src/modules/common/less/form.less

```diff
@@ -0,0 +1,277 @@
+@import (reference) '~assets/styles/shared';
+
+.Form {
+  margin: auto;
+  max-width: 62.75rem;
+  padding: 3.75rem 0 9.125rem;
+}
+
+.Form__error {
+  background-color: @color-purple;
+  border-radius: 3px;
+  color: @color-white;
+  font-size: 0.625rem;
```

Comment:
I’ve noticed a few locations where font size is being explicitly defined.  All occurrences should be using the shared font-size vars.  While not as rigid, line-height (assuming congruence to font size), should also use the vars for easier refactors to sizing going forward.